packages/dapp-svelte-wallet/api/test/test-lib-wallet.js

```diff
@@ -14,9 +14,9 @@ import { makeRegistrar } from '@agoric/registrar';
 import { assert } from '@agoric/assert';
 import { E } from '@agoric/eventual-send';
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { makeBoard } from '@agoric/cosmic-swingset/lib/ag-solo/vats/lib-board';
+import { makeBoard } from '@agoric/vats/src/lib-board';
```

Comment:
I guess part of the process towards removing `@agoric/vats` will be to split each vat out into a separate package, or maybe into groups of vats that e.g. are all likely to be used by a dapp unit test. So I can envision future improvements here.

Reply:
I suspect that `@agoric/vats` will be around until after we've got a proper `ag-solo`, or until we have to port our platform to another chain, whichever comes first.